### DISCUSSION TITLE
Seeking code reviewers!

### DISCUSSION THREAD

**Discussion by NiklasGustafsson** [xamarin/TorchSharp#513] [Announcements]
Hello,

We're looking for people who are interested in helping the development of TorchSharp by making themselves available to do code reviews of pull requests every now and then.

Please respond to this discussion if you are willing to help out with this.

Thanks,

Niklas

**Comment by oluwandabira** [comment]
I'm willing to give it a shot, although I don't have much experience with the code base yet.

**Comment by lostmsu** [comment]
I'm in

**Comment by NiklasGustafsson** [comment]
Okay, I've added you to the repo so that your names will appear on the list of code reviewers. I think you have to accept the invitation before you will actually show up.

**Comment by NiklasGustafsson** [comment]
@lostmsu. @oluwandabira -- apparently, I violated some sort of policy by adding you as individual members, rather than via a team. I'm going to have to figure out the right way to do it. In the meantime, I won't be able to use the GH UI to add you as reviewers, I'll just ask informally for a review.
